src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -22,62 +22,101 @@
 import java.util.stream.Collectors;
 
 public class Statistics {
-  private static final double LOWER_END_VAL = -1.0;
-  private static final double UPPER_END_VAL = 1.0;
-  private static final BigDecimal INTERVAL = BigDecimal.valueOf(0.2);
-  private static final BigDecimal UPPER_END = BigDecimal.valueOf(UPPER_END_VAL);
-  private static final BigDecimal LOWER_END = BigDecimal.valueOf(LOWER_END_VAL);
+  private static final double LOWER_SCORE_VAL = -1.0;
+  private static final double UPPER_SCORE_VAL = 1.0;
+  private static final double LOWER_MAGNITUDE_VAL = 0.0;
+  private static final double UPPER_MAGNITUDE_VAL = 2.0;
```

Comment:
Since you mentioned only a few comments got up to 2.0, maybe we can make this a little bigger just in case (4.0?)

Reply:
Update: test the code and 90% of scores are within ranges btw 0 - 0.2